doc/nrf/samples/wifi_zephyr.rst

```diff
@@ -0,0 +1,48 @@
+.. _wifi_zephyr_samples:
+
+Wi-Fi: Zephyr networking samples
+################################
+
+In addition to nRF Connect SDK samples, it is also possible to run selected Zephyr networking
```

Comment:
should we mention "upstream Zephyr" samples here, to indicate that these are maintained in the upstream project?
e.g. "it is also possible to run selected TCP/IP networking samples provided and maintained as part of  the upstream Zephyr project"?

Reply:
I'll update this section, however I'd drop the `TCP/IP` part and stick to `networking samples`. IMO it's narrowing too much, we also support CoAP/DNS etc which are UDP.

Reply:
That's fine. For me TCP/IP stack refers to the stack model name, not just the two protocols (i.e. it is another way to refer to the internet protocol suite)